doc/api/fs.md

```diff
@@ -574,6 +574,9 @@ changes:
     description: The `path` parameter can be a WHATWG `URL` object using `file:`
                  protocol. Support is currently still *experimental*.
 -->
+Note that this is a blocking call. Avoid this method in an asynchronous
+environment. See [Overview of Blocking vs Non-Blocking](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking).
```

Comment:
`blocking).` <- this period seems erroneous here and in all the other places.

Reply:
And maybe it is worth to use a bottom reference instead of repeating the link URL: this will reduce the file size and the length of the lines.

Reply:
Thanks I am fixing it now.